.gitignore

```diff
@@ -16,11 +13,6 @@ hs_err_pid*
 ## IntelliJ
 ##############################
 out/
-.idea/*
-.idea_modules/
-*.iml
-*.ipr
-*.iws
```

Comment:
After I push everything, when I try to clone and run the application, it wasn't working. So I thought it might be about files in the .idea. What about .iml file? Should I push it?